common/fixation_entry.go

```diff
@@ -164,12 +164,12 @@ func (fs *FixationStore) getEntryStore(ctx sdk.Context, index string) *prefix.St
 	return &store
 }
 
-func (fs *FixationStore) replaceTimer(ctx sdk.Context, prev, next types.Entry, kind byte) {
+func (fs *FixationStore) replaceTimer(ctx sdk.Context, prev, next types.Entry, block uint64, kind byte) {
```

Comment:
it would be helpful if you put a comment on how to use this function and its purpose (if I understand correctly it's not a replacement as much as a timer transfer. You delete the timer for the prev entry in a certain height and assign a new timer for the next entry. Did I understand correctly)?